OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessageRepository.java

```diff
@@ -75,6 +75,7 @@ synchronized void saveInAppMessage(OSInAppMessage inAppMessage) {
         values.put(OneSignalDbContract.InAppMessageTable.COLUMN_NAME_DISPLAY_QUANTITY, inAppMessage.getDisplayStats().getDisplayQuantity());
         values.put(OneSignalDbContract.InAppMessageTable.COLUMN_NAME_LAST_DISPLAY, inAppMessage.getDisplayStats().getLastDisplayTime());
         values.put(OneSignalDbContract.InAppMessageTable.COLUMN_CLICK_IDS, inAppMessage.getClickedClickIds().toString());
+        values.put(OneSignalDbContract.InAppMessageTable.COLUMN_DISPLAYED, inAppMessage.isDisplayed() ? 1 : 0);
```

Comment:
1. No, this will be merged all together so it shouldn't
2. https://www.sqlite.org/datatype3.html  "SQLite does not have a separate Boolean storage class. Instead, Boolean values are stored as integers 0 (false) and 1 (true)."
I will refactor to save as boolean but the parsing will stay the same

Reply:
1. Already covered by the `upgradeToV7` function since it is a totally new table
2. Yup, integer is as close as you get to a boolean.